parlai/tasks/dstc7/agents.py

```diff
@@ -96,6 +99,17 @@ def share(self):
         return shared
 
 
+class DSTC7TeacherAugmented(DSTC7Teacher):
+    """
+    Augmented.
```

Comment:
can you add more detail in this docstring? what does augmented mean? what is the difference between augmented and augmented sampled?

Reply:
to be honest, @samhumeau would have a better idea of what exactly this means. I will consult him offline (unless he wants to comment here 😄)